radixdlt-java/radixdlt-cli/src/main/java/com/radixdlt/cloud/AWSSecrets.java

```diff
@@ -277,4 +280,18 @@ private static boolean canBeUpdated(final AWSSecretsOutputOptions awsSecretsOutp
 			&& (!awsSecretsOutputOptions.getNetworkName().equalsIgnoreCase("betanet")
 					|| !awsSecretsOutputOptions.getNetworkName().equalsIgnoreCase("mainnet"));
 	}
+
+	private static String findNetworkId(String networkName) {
+		Map<String, String> networks = new HashMap<String, String>();
+		networks.put("releasenet", "3");
+		networks.put("rcnet", "4");
+		networks.put("milestonenet", "5");
+		networks.put("devopsnet", "6");
```

Comment:
This info should be in `Network.java` which @siy should be exposing in radixdlt-common